### DISCUSSION TITLE
Power app license needed for basic users

### DISCUSSION THREAD

**Discussion by sacha-rk** [microsoft/Templates-for-Power-Platform#66] [General]
Hi,
I installed the app in a development environment, and everything is working as expected. However, when a "basic user" tries to access the app, it states that a Power Apps license is required.
Could you confirm whether a Power Apps license is necessary for using (not administering) the Kudo app?
Thanks.

**Comment by mhuguet** [comment]
Each of the app templates requires premium licensing to use. Premium features are required for each of the templates.
